model/plugins/flush_cache/operations.smithy

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+use opensearch.openapi#vendorExtensions
+
+@externalDocumentation(
+    "API Reference": "https://opensearch.org/docs/2.7/security/access-control/api/#flush-cache"
+)
+
+@vendorExtensions(
+    "x-operation-group": "flush_cache",
```

Comment:
remember to put this operation in security namepsace. Same with other operations.

Reply:
> remember to put this operation in security namepsace. Same with other operations.

Yeap sure, I'm sorry that I missed that